cpr/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 message(STATUS "Using CURL_INCLUDE_DIRS: ${CURL_INCLUDE_DIRS}.")
 
-add_library(${CPR_LIBRARIES}
+if($(BUILD_SHARED_LIBS))
+    set(SHARED SHARED)
+endif()
+
+add_library(${CPR_LIBRARIES} ${SHARED}
```

Comment:
This can be removed, since CMake does use BUILD_SHARED_LIBS if no build type is specified see [docs](https://cmake.org/cmake/help/v3.0/variable/BUILD_SHARED_LIBS.html).

Reply:
I also think that this is a standard feature from cmake, so not really needed